docs/learning-resources.md

```diff
@@ -26,3 +26,9 @@ community, file an issue or submit a pull request.
 
 * [Bootstrap a Zero Configuration React App with Neutrino](https://egghead.io/lessons/react-bootstrap-a-zero-configuration-react-app-with-neutrino) by [@markshust](https://twitter.com/markshust)
 * [Empower and Simplify JavaScript Development](https://www.youtube.com/watch?v=fz5jMdmKmRI) by [@eliperelman](https://twitter.com/eliperelman) at [@nebraskajs](https://nebraskajs.com)
+
+## Thirdparty Presets or Middleware
```

Comment:
Can you make "Thirdparty" two words, e.g., "Third Party"? Also change the word "or" to "and".

Reply:
@eliperelman Done!